### ISSUE TITLE
OM suggests longer route for cycling

### ISSUE THREAD

**Issue by Maschga** [organicmaps/organicmaps#9131]
OM suggests this route for cycling when routing from [a street in a village](https://www.openstreetmap.org/#map=18/49.424249/12.099000&layers=N) to [this house](https://www.openstreetmap.org/way/945294437#map=19/49.418725/12.114452&layers=N):
![bad_route](https://github.com/user-attachments/assets/4916f509-f266-415e-b50f-54a29c627485)
[See Route on OSM](https://www.openstreetmap.org/directions?engine=fossgis_valhalla_bicycle&route=49.42426%2C12.09901%3B49.41890%2C12.11393#map=16/49.42197/12.10696&layers=N)

This is a street with much traffic and without a cycle path making it uncomfortable to cycle.

**Expected behaviour**
Instead OM should suggest this route:
![improved_route](https://github.com/user-attachments/assets/3f22c315-9f37-4306-8799-a81cb3ff1943)
[See Route on OSM](https://www.openstreetmap.org/directions?engine=fossgis_osrm_bike&route=49.42426%2C12.09901%3B49.41890%2C12.11393)

This path is shorter (200m) and, according to OSM, also asphalted.

**System information:**
 - Operating system and its version: Android 13
 - Organic Maps version: 2024.08.17-4-Google
 - Device Model: Samsung A71

**Comment by Maschga** [comment]
I also have the feeling that OM's routing is very much based on the route from the car. Could it be that field and forest paths are not popular for bicycle routing? I understand that for racing bikes, but a mountain bike can also be used on a compacted or worse path.

**Comment by muralito** [comment]
Maybe it's a problem with the data.
This way https://www.openstreetmap.org/way/30074328 is highway=track 
Are you sure this is correct according to https://wiki.openstreetmap.org/wiki/Tag:highway%3Dtrack ?

**Comment by Maschga** [comment]
Good idea. According to the wiki, these streets should rather be tagged as `highway=unclassified`.

**Comment by Maschga** [comment]
Tagging with `highway=unclassified` fixed the issue. Thank you!
